manifest.json

```diff
@@ -65,16 +65,56 @@
     "metadata_filename": "realm",
     "file_format": "json"
   },
-  "react-native-ts.todo.flex": {
+  "react-native.todo.v2": {
```

Comment:
Ah! I didn't realize we were updating the names of these. When we had talked about this before, we were going to leave the names the same so this would be a drop-in replacement for V1. 

Is there a reason we're making this a v2 instead of just replacing? They run against a different backend, so I think most people are just going to want "the client" that matches the Device Sync app they create in the UI. Having to think about `react-native-todo.flex` vs `react-native.todo.v2` to decide which one to get (if they do it via CLI) is maybe an unnecessary point of friction?

Reply:
No, per the release plan, this is a temporary ID so we can test on the CLI without disrupting v1 - we will update it to be a drop-in replacement later.